pkg/networkserver/errors.go

```diff
@@ -19,52 +19,53 @@ import (
-	errInvalidNwkSEncKey         = errors.DefineInvalidArgument("invalid_nwk_s_enc_key", "invalid NwkSEncKey")
-	errInvalidPayload            = errors.DefineInvalidArgument("payload", "invalid payload")
-	errInvalidRx2DataRateIndex   = errors.DefineInvalidArgument("rx2_data_rate_index", "invalid Rx2 data rate index")
-	errInvalidSNwkSIntKey        = errors.DefineInvalidArgument("invalid_s_nwk_s_int_key", "invalid SNwkSIntKey")
-	errJoinServerNotFound        = errors.DefineNotFound("join_server_not_found", "Join Server not found")
-	errMACRequestNotFound        = errors.DefineInvalidArgument("mac_request_not_found", "MAC response received, but corresponding request not found")
-	errNoFrequencyPlan           = errors.DefineInvalidArgument("no_frequency_plan", "no frequency plan specified")
-	errNoDevEUI                  = errors.DefineInvalidArgument("no_dev_eui", "no DevEUI specified")
-	errNoJoinEUI                 = errors.DefineInvalidArgument("no_join_eui", "no JoinEUI specified")
-	errNoPath                    = errors.DefineNotFound("no_downlink_path", "no downlink path available")
-	errNoPayload                 = errors.DefineInvalidArgument("no_payload", "no message payload specified")
-	errNoRekey                   = errors.DefineInvalidArgument("no_rekey", "rekey not received after join-accept")
-	errOutdatedData              = errors.DefineNotFound("outdated_data", "data is outdated")
-	errRawPayloadTooShort        = errors.Define("raw_payload_too_short", "length of RawPayload must not be less than 4")
-	errSchedule                  = errors.Define("schedule", "all downlink scheduling attempts failed")
-	errScheduleTooSoon           = errors.DefineUnavailable("schedule_too_soon", "confirmed downlink is scheduled too soon")
-	errUnknownBand               = errors.Define("unknown_band", "band is unknown")
-	errUnknownChannel            = errors.Define("unknown_chanel", "channel is unknown")
-	errUnknownFNwkSIntKey        = errors.DefineNotFound("unknown_f_nwk_s_int_key", "FNwkSIntKey is unknown")
-	errUnknownFrequencyPlan      = errors.Define("unknown_frequency_plan", "frequency plan is unknown")
-	errUnknownMACState           = errors.DefineFailedPrecondition("unknown_mac_state", "MAC state is unknown")
-	errUnknownNwkSEncKey         = errors.DefineNotFound("unknown_nwk_s_enc_key", "NwkSEncKey is unknown")
-	errUnknownSNwkSIntKey        = errors.DefineNotFound("unknown_s_nwk_s_int_key", "SNwkSIntKey is unknown")
-	errUnsupportedLoRaWANVersion = errors.DefineInvalidArgument("unsupported_lorawan_version", "unsupported LoRaWAN version: {version}", "version")
-	errUplinkChannelNotFound     = errors.DefineNotFound("uplink_channel_not_found", "uplink channel not found")
-	errUplinkNotFound            = errors.DefineNotFound("uplink_not_found", "uplink not found")
+	errApplicationDownlinkTooLong = errors.DefineInvalidArgument("application_downlink_too_long", "application downlink payload is too long")
```

Comment:
Unfortunate formatting diff. This is the only new line I added here.

Reply:
This kind of stuff breaks our blames. Not critical for here, but this is one of the reasons why I typically declare errors close to where I use them